### PR TITLE
Qa/UI driver setup

### PR DESCRIPTION
**Pull request by PoroQa**
Created Driver setup class, Driver type enum class and Element control class with wrapper methods for controlling web elements

### CODE REVIEW

**Review by JadrankoH — APPROVED** [review]

File: QA/PoroDevTest/src/main/java/common/ui_setup/DriverSetup.java [hunk 1]
```diff
@@ -1,4 +1,60 @@
...
+import org.openqa.selenium.firefox.FirefoxDriver;
+import java.time.Duration;
+
 public class DriverSetup {
+    private WebDriver driver;
+    private int pageLoadTime;
+    private int elementDetectionTimeout;
+
+    public DriverSetup(DriverType browserType) {
+        pageLoadTime = 10;
```
Comment (udrakulic-htec) [inline QA/PoroDevTest/src/main/java/common/ui_setup/DriverSetup.java]: Initialization of pageLoad and timeout can be inside constants class

**Review by udrakulic-htec — CHANGES_REQUESTED** [review]

File: QA/PoroDevTest/src/main/java/common/ui_setup/DriverSetup.java (hunk 1 above)
Comment (udrakulic-htec) [reply QA/PoroDevTest/src/main/java/common/ui_setup/DriverSetup.java]: Sample UI test class should be added to UI setup PR

Comment (PoroQa) [reply QA/PoroDevTest/src/main/java/common/ui_setup/DriverSetup.java]: Constant class is created

File: QA/PoroDevTest/src/test/java/TestSetup/SampleTestSetup.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    public void setupBeforeTestClass() {
+        System.out.println("This will trigger first");
+        driverSetup = new DriverSetup(DriverType.CHROME);
+    }
+
+    @BeforeMethod
+    @Override
+    public void setupBeforeMethod() {
+        System.out.println("This will trigger second after test class method");
+        driverSetup.navigateToUrl("https://www.google.rs/webhp?source=search_app&gws_rd=cr,ssl&ei=zzA0VO2GE4n_ygP1l4GAAw");
```
Comment (udrakulic-htec) [inline QA/PoroDevTest/src/test/java/TestSetup/SampleTestSetup.java:26]: In some future commit pls add just navigation to https://www.google.rs

**Review by udrakulic-htec — APPROVED** [review]

File: QA/PoroDevTest/src/test/java/TestSetup/SampleTestSetup.java (hunk 2 above)
Comment (PoroQa) [reply QA/PoroDevTest/src/test/java/TestSetup/SampleTestSetup.java:26]: Got it

**Pull request merged**